From 31be225e30b0b2ef619fd2140713451273830a60 Mon Sep 17 00:00:00 2001
From: Alexander Tsidaev <none@none.no>
Date: Mon, 28 Nov 2022 01:06:43 +0500
Subject: [PATCH] Fix baudrate for FT232RL

---
 FlashGBX/hw_GBxCartRW_ofw.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/FlashGBX/hw_GBxCartRW_ofw.py b/FlashGBX/hw_GBxCartRW_ofw.py
index 5930771..3b395b2 100644
--- a/FlashGBX/hw_GBxCartRW_ofw.py
+++ b/FlashGBX/hw_GBxCartRW_ofw.py
@@ -138,7 +138,7 @@ class GbxDevice:
 	POS = 0
 	NO_PROG_UPDATE = False
 	FAST_READ = False
-	BAUDRATE = 1000000
+	BAUDRATE = 150000
 	
 	def __init__(self):
 		pass
@@ -154,7 +154,7 @@ class GbxDevice:
 		else:
 			comports = serial.tools.list_ports.comports()
 			for i in range(0, len(comports)):
-				if comports[i].vid == 0x1A86 and comports[i].pid == 0x7523:
+				if comports[i].vid == 0x0403 and comports[i].pid == 0x6001:
 					ports.append(comports[i].device)
 					#break
 		if len(ports) == 0: return False
-- 
2.11.1.windows.1

